packages/app-root/src/components/RootLayout.js

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import { Grommet } from 'grommet'
+import zooTheme from '@zooniverse/grommet-theme'
+import { base as baseTheme } from 'grommet'
+import { ZooHeader, ZooFooter } from '@zooniverse/react-components'
+
+const theme = { ...baseTheme, ...zooTheme }
+
+export default function RootLayout({ children }) {
+  return (
+    <Grommet
+      background={{
+        dark: 'dark-1',
+        light: 'light-1'
+      }}
+      theme={theme}
+    >
+      <ZooHeader />
+      <main>{children}</main>
```

Comment:
Longer term, inserting `main` here prevents child pages from having their own headers and sidebars.

Reply:
Do you recommend moving `main` to page.js (Zooniverse homepage), as well as all of the other page.js files?

Reply:
I think so. If you look at projects, there's a project header and project navigation, then a main section with an optional aside. 'About' pages have their own header and sidebar too, before you get to the main content.

Reply:
Good point!